include/invert_quda.h

```diff
@@ -432,7 +432,7 @@ namespace quda {
     /**
        @brief Naive loop over RHS, for solvers that are not yet multi-RHS aware
      */
-    virtual void operator()(cvector_ref<ColorSpinorField> &out, cvector_ref<ColorSpinorField> &in)
+    void operator()(cvector_ref<ColorSpinorField> &out, cvector_ref<ColorSpinorField> &in)
```

Comment:
Just trying to understand the change here: why do we need to remove the `virtual`?

Reply:
It's a dumb (as far as I'm concerned) warning by gcc 13.  It raises a warning if we have multiple virtual polymorphic functions in the same class hierarchy but we don't specialize all of them.  This is a just a quick fix to silence the warning, the real fix will come when we have MRHS support for all solvers as opposed to this wrapper.

Reply:
Is it called [gcc Woverloaded-virtual](https://stackoverflow.com/questions/9995421/gcc-woverloaded-virtual-warnings)?

Reply:
I guess we only use the base class pointer so it is not an issue for us right?

Reply:
Yeah it's not an issue at all.  Just a quick patch to silence the warning until I add proper MRHS solvers.